apps/framework-cli/src/cli.rs

```diff
@@ -152,6 +161,37 @@ async fn top_command_handler(settings: Settings, commands: &Commands) {
                 controller.add_routine(Box::new(CleanProject::new(project_arc, run_mode)));
                 controller.run_routines(run_mode);
             }
+            Commands::Docker { sub_method } => match sub_method.as_str() {
+                "init" => {
+                    let run_mode = RunMode::Explicit {};
+                    info!("Running docker init command");
+                    let project = Project::load_from_current_dir()
+                        .expect("No project found, please run `moose init` to create a project");
+                    let project_arc = Arc::new(project);
+                    let mut controller = RoutineController::new();
+                    controller.add_routine(Box::new(CreateDockerfile::new(project_arc)));
```

Comment:
Is the user supposed to edit this file? If yes then maybe it shouldn't be in the internal directory, if no then we can write it every time before we `build`.

Maybe the answer is "not really but here's an escape hatch if the user really wants to edit the file."

Reply:
Definitely something we can keep an eye on, in case the use-case emerges.  But I do see your  point. Good catch!